### ISSUE TITLE
[BUG] Config can not be opened

### ISSUE THREAD

**Issue by Auxiliotest** [arkayenro/arkinventory#1675]
**What version number you are using?**
3.10.06

**What game client are you playing? Retail, Burning Crusade, Classic / Live, PTR, Beta**
Retail

**What language is the game client set to?**
English

**Describe the bug**
Can't open config.

**To Reproduce**
Clicking the config option does nothing

**Screenshots**

**Additional context**
1x ...ceConfig/AceConfigDialog-3.0-86/AceConfigDialog-3.0.lua:1854: AceConfigRegistry-3.0-21:ValidateOptionsTable(): ArkInventory.args: expected a table, got 'nil'
[string "=[C]"]: ?
[string "@Interface/AddOns/ArkInventory/Externals/Ace3/AceConfig/AceConfigRegistry-3.0-21/AceConfigRegistry-3.0.lua"]:50: in function <...nfig/AceConfigRegistry-3.0/AceConfigRegistry-3.0.lua:45>
[string "@Interface/AddOns/ArkInventory/Externals/Ace3/AceConfig/AceConfigRegistry-3.0-21/AceConfigRegistry-3.0.lua"]:211: in function <...nfig/AceConfigRegistry-3.0/AceConfigRegistry-3.0.lua:206>
[string "@Interface/AddOns/ArkInventory/Externals/Ace3/AceConfig/AceConfigRegistry-3.0-21/AceConfigRegistry-3.0.lua"]:248: in function <...nfig/AceConfigRegistry-3.0/AceConfigRegistry-3.0.lua:220>
[string "@Interface/AddOns/ArkInventory/Externals/Ace3/AceConfig/AceConfigRegistry-3.0-21/AceConfigRegistry-3.0.lua"]:283: in function `ValidateOptionsTable'
[string "@Interface/AddOns/ArkInventory/Externals/Ace3/AceConfig/AceConfigRegistry-3.0-21/AceConfigRegistry-3.0.lua"]:325: in function `app'
[string "@Interface/AddOns/ArkInventory/Externals/Ace3/AceConfig/AceConfigDialog-3.0-86/AceConfigDialog-3.0.lua"]:1854: in function `Open'
[string "@Interface/AddOns/ArkInventory/Core/ArkInventoryConfig.lua"]:31: in function `Frame_Config_Show'
[string "@Interface/AddOns/ArkInventory/Core/ArkInventoryMenu.lua"]:125: in function `func'
[string "@Interface/AddOns/ArkInventory/Libs/ArkDewdrop/ArkDewdrop.lua"]:731: in function <...e/AddOns/ArkInventory/Libs/ArkDewdrop/ArkDewdrop.lua:670>

Locals:
(*temporary) = "AceConfigRegistry-3.0:ValidateOptionsTable(): ArkInventory.args: expected a table, got 'nil'"

**Comment by arkayenro** [comment]
thats typically from opening the config before the mount data is ready.

give it a bit of time to load and then load the config.

i've added some extra checking there so it shouldnt fail next release

**Comment by Auxiliotest** [comment]
Ok thx will try!

**Comment by Auxiliotest** [comment]
How long should it need, I now waited 5 mins but still got the error. Thx again

**Comment by Neischa** [comment]
Same issue here, after 2 hours in game still can not open config:(

**Comment by StefanLE1979** [comment]
same issue, slightly different error message:

```
Message: ...ace/AddOns/ArkInventoryConfig/ArkInventoryConfig.lua:25: attempt to index field 'SORTBY' (a nil value)
Time: Sat Nov 19 10:33:51 2022
Count: 1
Stack: ...ace/AddOns/ArkInventoryConfig/ArkInventoryConfig.lua:25: attempt to index field 'SORTBY' (a nil value)
[string "@Interface/AddOns/ArkInventoryConfig/ArkInventoryConfig.lua"]:25: in main chunk
[string "=[C]"]: in function `LoadAddOn'
[string "@Interface/AddOns/ArkInventory/Core/ArkInventory.lua"]:11228: in function `LoadAddOn'
[string "@Interface/AddOns/ArkInventory/Core/ArkInventoryConfig.lua"]:29: in function `Frame_Config_Show'
[string "@Interface/AddOns/ArkInventory/Core/ArkInventoryConfig.lua"]:68: in function <...face/AddOns/ArkInventory/Core/ArkInventoryConfig.lua:67>
[string "=(tail call)"]: ?
[string "@Interface/AddOns/Altoholic/Libs/AceConfig-3.0/AceConfigCmd-3.0/AceConfigCmd-3.0.lua"]:136: in function <.../AceConfig-3.0/AceConfigCmd-3.0/AceConfigCmd-3.0.lua:126>
[string "@Interface/AddOns/Altoholic/Libs/AceConfig-3.0/AceConfigCmd-3.0/AceConfigCmd-3.0.lua"]:401: in function <.../AceConfig-3.0/AceConfigCmd-3.0/AceConfigCmd-3.0.lua:327>
[string "=(tail call)"]: ?
[string "@Interface/AddOns/Altoholic/Libs/AceConfig-3.0/AceConfigCmd-3.0/AceConfigCmd-3.0.lua"]:762: in function `HandleCommand'
[string "@Interface/AddOns/Altoholic/Libs/AceConfig-3.0/AceConfigCmd-3.0/AceConfigCmd-3.0.lua"]:774: in function `?'
[string "@Interface/FrameXML/ChatFrame.lua"]:5224: in function `ChatEdit_ParseText'
[string "@Interface/FrameXML/ChatFrame.lua"]:4888: in function `ChatEdit_SendText'
[string "@Interface/FrameXML/ChatFrame.lua"]:4924: in function `ChatEdit_OnEnterPressed'
[string "*ChatFrame.xml:127_OnEnterPressed"]:1: in function <[string "*ChatFrame.xml:127_OnEnterPressed"]:1>
```

**Comment by arkayenro** [comment]
fixed in 3.10.07

**Comment by Auxiliotest** [comment]
Confirmed working fine in 3.10.07, thanks for the quick fix! :)
